src/Applications/GEOSdas_App/fvsetup

```diff
@@ -3499,42 +3496,54 @@ sub set_newradbc {
 }
 #=======================================================================
 sub set_ldasANA {
+    my ($dflt, $ans1, $ans2);
+    my (@lfo_files);
```

Comment:
I recognize `lfo_files` is just a variable name, but for consistency it may be better to use `bkglfo_files`

Reply:
will modify the name.